src/routes/formsgSiteLaunch.ts

```diff
@@ -194,14 +211,24 @@ export class FormsgSiteLaunchRouter {
     repoName: string,
     submissionId: string,
     domainValidationSource: string,
-    domainValidationTarget: string
+    domainValidationTarget: string,
+    primaryDomainSource: string,
+    primaryDomainTarget: string,
+    redirectionDomainSource?: string,
+    redirectionDomainTarget?: string
```

Comment:
hmm, this api is rather onerous, are we guaranteed that this is always present in an error state (if so, please ignore)

Reply:
This is the intended design! there should be no sending of verification details if _**any**_ of those fields are not present!

Reply:
^sure, but then should `redirectionDomain<Source | Target>` be optional if that's the case?

Reply:
yup, not all sites have `redirectionDomain<Source | Target>`

Reply:
sure! as an aside, if we want to guarantee that `redirectionDomain<Source | Target>` is either all provided, or not at all (ie, caller has to give both or nothing, cannot just give 1), might wanna consider using an object argument but otherwise, it's ok

Reply:
2dd9dfd